src/__tests__/ActionMenu2.test.tsx

```diff
@@ -140,5 +140,6 @@ describe('ActionMenu', () => {
     cleanup()
   })
 
-  checkStoriesForAxeViolations('ActionMenu2')
+  checkStoriesForAxeViolations('ActionMenu2/fixtures')
```

Comment:
This `fixtures`/`examples` split is interesting. Could we draft an ADR or some documentation explaining how and why we would use this pattern?

Reply:
Can do!